homeassistant/components/alarm_control_panel/reproduce_state.py

```diff
@@ -38,6 +41,9 @@
     STATE_ALARM_ARMED_VACATION,
     STATE_ALARM_DISARMED,
     STATE_ALARM_TRIGGERED,
+    STATE_ALARM_PENDING,
```

Comment:
Please don't change the entity integration in the same PR as another integration is changed.

Reply:
Removed the change, and submitted https://github.com/home-assistant/architecture/discussions/1125